articles/feature-engineering-in-machine-learning/index.md

```diff
@@ -0,0 +1,167 @@
+
+#### Features and Techniques
+We have defined what a feature is. But, for a wholesome understanding, it is important that we delve deeper to understand what kinds of features exist. After that, we finally will go over a few feature engineering techniques.
+
+Below are three general types of features.
+
+##### Examples of Features
+- **Categorical features**. These are features derived from categorical data. Categorical data is data that is grouped into categories. It is often in a non-numerical format. An example of such data could be, a type of pet. This could be a dog, cat, snake, hamster and so forth.
+
+- **Text features**. Text features are derived from text data. This means that text is encoded to corresponding numerical values.
+
+- **Image features**. Images are converted into suitable formats for analysis. The result is image-based features.
+
+Understanding of features puts us in a better position to dive into feature engineering techniques.  
+
+Let’s briefly define 5 techniques.
+
+##### 5 Feature Engineering Techniques
+###### Imputation
+It is common to deal with incomplete datasets. As we mentioned previously, feature engineering involves handling missing values. The technique responsible for this is imputation.
+
+###### Outliers
+Outliers are datapoints observed to be too far from the rest. They distort the results of the models therefore they must be handled. This technique deals with first identifying then trimming the outliers.
+
+###### Log Transform
+Skewness refers to the measure of how asymmetric a data distribution can be. When a dataset is skewed, it impacts performance of a model. Log transform comes in to fix skewness of a dataset. It has a purpose to make the dataset distribution as close to normal as possible. This technique also reduces the impact of outliers. It is worth noting that this technique works only with positive values.
+
+###### Binning
+Overfitting refers to a model that contains more parameters than can be accounted for by dataset. Noisy data contributes to overfitting. As a result, we use a binning technique to smooth noisy data. This involves dividing features into different bins. These features may be numerical or categorical.
+
+###### Feature Split
+This simply refers to splitting of a feature into two or more parts. This may be done to create new features or to aid an algorithm to get a better understanding of the dataset.
+
+#### Feature Engineering vs Feature Selection
+We have already defined and explored the feature engineering process. Nonetheless, it is common to have newbies use feature engineering and feature selection interchangeably.
```

Comment:
Restructure the second sentence. Avoid using words like newbies since this is an introductory article. We want to avoid offending anyone reading the article.